src/ui/server/content-strings/fields/insurance/your-business/index.ts

```diff
@@ -62,6 +64,24 @@ export const FIELDS = {
       },
     },
   },
+  CONTACT: {
+    [CONTACT_COMPANY_NAME]: {
+      LABEL: 'Your company name',
+      HINT: 'This information comes from Companies House',
+    },
+    [FIRST_NAME]: {
+      LABEL: 'First name',
+    },
+    [LAST_NAME]: {
+      LABEL: 'Last name',
+    },
+    [EMAIL_ADDRESS]: {
+      LABEL: 'Email address',
+    },
```

Comment:
Sorry to touch on this again but the labels are now duplicated, I think we should extract first/last/email fields in content-strings so they're at the root of `ACCOUNT_FIELDS` then we can reuse everywhere and be DRY. What do you think? Can do after this PR